che-calculator/mccabe-thiele-binary-distillation.html

```diff
@@ -158,26 +159,31 @@ <h3>McCabe-Thiele Diagram</h3>
             There are several components in a McCabe-Thiele Diagram:
             <h5>x-y diagram</h5>
             The x-y diagram depicts vapor-liquid equilibrium data of the binary components, which any points on the graph shows the amount of liquid that is equilibrium 
-            with vapor at certain temperature. The diagram will help to determine the relative volatility (\(\alpha\)) to predict the amount of separation possible. 
-            When \(\alpha &lt 1.05\), the separation process will take more trays as the components are difficult to seperate using this method. Relative volatility is as follows:
+            with vapor at certain temperature. This graph is used to calculate relative volatility (\(\alpha\)) to determine the amount of separation possible. Relative volatility
+            is as follows:
             \[\alpha= {{y_i/x_i}\over{y_j/x_j}}\]
             Where \(y\) is the vapor fraction, \(x\) is the liquid fraction, and \(i, j\) is the chemical components. In the binary system, the above equation can be expressed as
             \[\alpha= {{y_i/x_i}\over{(1-y_i)/(1-x_i)}}\]
             <br>
-            <h5>Operating line</h5>
-            Operating line is a simple line equation to determine amount of liquid per amount of vapour (\( \mathcal{L} / \mathcal{V}\)) in the distillation process.
+            For bubble point calculations which vapor-phase compositions are unknown, we can simplify the calculations by asumming \(\sum y_i=1\), so that the Raoult's law become
```

Comment:
line 168 until 180 is obsolete. Please use the new one.

Reply:
done, you can check it aagain :D